judge-control-app/src/remote_exec/ssh/tests.rs

```diff
@@ -0,0 +1,137 @@
+#![allow(clippy::unwrap_used)]
+use super::super::{ssh::SshConnection, traits::RemoteExec};
+use anyhow::Result;
+use std::time::Duration;
+use uuid::Uuid;
+
+#[tokio::test]
+async fn test_ssh_connection() {
+    let uuid = Uuid::new_v4();
+    activate_container(uuid).await;
+    let mut ssh = SshConnection {
+        addrs: "localhost:2022",
+        username: "root".to_string(),
+        password: "password".to_string(),
+    };
+    let resp = ssh
+        .exec(
+            "cat /flag",
+            Duration::from_secs(60),
+            Duration::from_secs(60),
+        )
+        .await;
+    stop_ssh_docker_container(uuid).await.unwrap();
+    assert!(resp.is_ok());
+    assert_eq!(resp.unwrap(), "TEST_FLAG\n");
+}
+
+#[tokio::test]
+async fn test_ssh_connection_timeout() {
+    let uuid = Uuid::new_v4();
+    activate_container(uuid).await;
+    let mut ssh = SshConnection {
+        addrs: "localhost:2022",
+        username: "root".to_string(),
+        password: "password".to_string(),
+    };
+    let resp = ssh
+        .exec(
+            "sleep 1",
+            Duration::from_millis(1),
+            Duration::from_millis(1),
+        )
+        .await;
+    stop_ssh_docker_container(uuid).await.unwrap();
+    assert!(resp.is_err_and(|e| match e {
+        super::SshExecError::RemoteServerError(_) => true,
+        _ => false,
+    }));
+    return;
+}
+
+async fn activate_container(uuid: Uuid) -> () {
```

Comment:
unneeded unit return type
```suggestion
async fn activate_container(uuid: Uuid) {
```